sdk/core/core-http/package.json

```diff
@@ -81,7 +82,7 @@
     "test": "npm run build:test && npm run unit-test && npm run integration-test",
     "unit-test": "npm run unit-test:node && npm run unit-test:browser",
     "unit-test:browser": "node ./node_modules/karma/bin/karma start karma.conf.ts --browsers ChromeNoSecurity --single-run",
-    "unit-test:node": "nyc mocha",
+    "unit-test:node": "cross-env TS_NODE_FILES=true nyc mocha",
```

Comment:
Interesting, what does `TS_NODE_FILES` do?

Reply:
@daviwil I had no idea until a few minutes ago. It affects the way ts-node reads tsconfig.json. It causes it to load the includes, excludes, and files (it apparently does not do so by default). I'm not 100% sure why it has this effect, but in this case it is required for ts-node to read the dom.d.ts file. Previously the unit-tests were getting DOM types from lib:dom, since `compilerOptions` is loaded by default.

Reply:
I googled the same problem and [came to the same conclusion](https://github.com/Azure/azure-sdk-for-js/commit/f310ec81ce3152968450c7e6e9ffd6e7c33cc819) some months ago, fwiw